test/flint/fmpz_mod_poly-test.jl

```diff
@@ -68,6 +68,10 @@
   Rx,  = polynomial_ring(R, "x")
   @test base_ring(Rx) === R
   @test Rx === polynomial_ring(R, "x")[1]
```

Comment:
I meant inserting something like this:
```suggestion
  @test Rx === polynomial_ring(R, "x")[1]
  @test krull_dim(Rx) == 2
```

Also, these tests should be in their own testset: `@testset "ZZModPolyRingElem.krull_dim" begin`